internal/bgp/frr/frr.go

```diff
@@ -239,8 +237,17 @@ func (sm *sessionManager) createConfig() (*frrConfig, error) {
 	return config, nil
 }
 
-func NewSessionManager() *sessionManager {
-	return &sessionManager{
-		sessions: map[string]*session{},
+var debounceTimeout = 500 * time.Millisecond
```

Comment:
const ?

Reply:
I left it as a var so we can put a lower variable in unit tests.
Otherwise unit tests execution would have take longer which is not desireable.